tests/contract_testcases/solana/selectors-must-be-different.dot

```diff
@@ -1,43 +1,43 @@
+	contract_15 [label="contract f\ntests/contract_testcases/solana/selectors-must-be-different.sol:26:1-33:2"]
+	f1_16 [label="function f1\ncontract: f\ntests/contract_testcases/solana/selectors-must-be-different.sol:29:2-22\nsignature f1()\nvisibility public\nmutability nonpayable\nselector 41424344"]
+	f2_17 [label="function f2\ncontract: f\ntests/contract_testcases/solana/selectors-must-be-different.sol:31:2-22\nsignature f2()\nvisibility public\nmutability nonpayable\nselector 41424344"]
+	f3_18 [label="function f3\ncontract: f\ntests/contract_testcases/solana/selectors-must-be-different.sol:32:2-22\nsignature f3()\nvisibility public\nmutability nonpayable"]
+	contract_19 [label="contract g\ntests/contract_testcases/solana/selectors-must-be-different.sol:35:1-40:2"]
+	f1_20 [label="function f1\ncontract: g\ntests/contract_testcases/solana/selectors-must-be-different.sol:36:2-22\nsignature f1()\nvisibility public\nmutability nonpayable"]
+	f3_21 [label="function f3\ncontract: g\ntests/contract_testcases/solana/selectors-must-be-different.sol:39:2-22\nsignature f3()\nvisibility public\nmutability nonpayable\nselector c27fc305"]
+	diagnostic [label="found abstract contract 'c'\nlevel Debug\ntests/contract_testcases/solana/selectors-must-be-different.sol:1:1-10:2"]
+	diagnostic_24 [label="function 'f2' selector is the same as function 'f1'\nlevel Error\ntests/contract_testcases/solana/selectors-must-be-different.sol:6:2-22"]
+	note [label="definition of function 'f1'\ntests/contract_testcases/solana/selectors-must-be-different.sol:3:2-22"]
+	diagnostic_26 [label="function 'f3' selector is the same as function 'f1'\nlevel Error\ntests/contract_testcases/solana/selectors-must-be-different.sol:9:2-22"]
+	note_27 [label="definition of function 'f1'\ntests/contract_testcases/solana/selectors-must-be-different.sol:3:2-22"]
+	diagnostic_28 [label="found contract 'd'\nlevel Debug\ntests/contract_testcases/solana/selectors-must-be-different.sol:12:1-18:2"]
+	diagnostic_29 [label="c is already defined as a contract name\nlevel Warning\ntests/contract_testcases/solana/selectors-must-be-different.sol:13:13-14"]
+	note_30 [label="location of previous definition\ntests/contract_testcases/solana/selectors-must-be-different.sol:1:1-10:2"]
+	diagnostic_31 [label="function 'f1' selector is the same as function 'c'\nlevel Error\ntests/contract_testcases/solana/selectors-must-be-different.sol:17:2-22"]
+	note_32 [label="definition of function 'c'\ntests/contract_testcases/solana/selectors-must-be-different.sol:13:13-14"]
+	diagnostic_33 [label="found contract 'e'\nlevel Debug\ntests/contract_testcases/solana/selectors-must-be-different.sol:20:1-24:2"]
+	diagnostic_34 [label="function 'f1' selector '[0x1]' must be 4 bytes rather than 1 bytes\nlevel Error\ntests/contract_testcases/solana/selectors-must-be-different.sol:22:2-16"]
```

Comment:
Shouldn't this be `0x01`?
Also, when I write the selector as `[1, 2, 3, 4, 255]`, will we print it as a hex array? I guess this might confuse developers.

Reply:
I've removed selector printing from the error. The location points to the selector already